modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6224,6 +6224,8 @@ disable-as-a-collection-provider-help=Disabling this as a collection provider wi
 disable-caching=Disable Caching
 disable-document-recording=Disable Document Recording
 disable-forwarding=Disable Forwarding
+disable-get-remote-methods=Disable getRemoteAddr() and getRemoteHost() methods
+disable-get-remote-methods-help=If checked, the methods getRemoteAddr() and getRemoteHost() will be undefined within the Liferay.ThemeDisplay global object. This configuration is primarily implemented to mitigate privacy risks by avoiding the exposure of client IP/host information and to optimize performance by allowing public caching of pages for unauthenticated users. Note that checking this option will make custom code that relies on these methods fail.
```

Comment:
We'd like to make it clear we mean `Liferay.ThemeDisplay`  javascript global object. I'd say most people knows this already but imo it won't hurt, as we are not saying it in the config section naming either